src/context.ts

```diff
@@ -38,8 +37,8 @@ export interface NotesContext extends ICommonContext {
 /** Studio */
 
 export interface IStudioConfigContext {
-	/** Returns a map of the studio configs */
-	getStudioConfig: () => Readonly<{ [key: string]: ConfigItemValue }>
+	/** Returns the Studio blueprint config */
```

Comment:
We should change the description to reference `BlueprintManifestType.parseConfig`

Reply:
I made the description mention `preprocessConfig`.